src/update-feeds-loops/update-feeds-loops.ts

```diff
@@ -331,7 +331,7 @@ export const processBatch = async (
       //
       // We can't differentiate between these cases unless we check recent update transactions, which we don't want to
       // do.
-      logger.debug(`Clearing data feed update timestamp because it no longer needs an update.`, {
+      logger.info(`Clearing data feed update timestamp because it no longer needs an update.`, {
```

Comment:
This should have been probably be INFO all along. This is not consistent with [this](https://github.com/api3dao/airseeker-v2/blob/0a06098b308e9bdb141718b1b1bdff76ed30f7e5/src/deviation-check/deviation-check.ts#L64) being an INFO level as well.